keps/sig-api-machinery/2885-server-side-unknown-field-validation/README.md

```diff
@@ -937,7 +907,37 @@ Why should this KEP _not_ be implemented?
 -->
 
 ## Alternatives
-* Content-Type Header vs Query Param (see [#proposal](#proposal))
+
+### Content-Type Header
```

Comment:
```suggestion
### HTTP header mechanism
```
One option is to send Content-Type; the other is to send `X-Kubernetes-Validation`.

Reply:
done